types/artifacts.d.ts

```diff
@@ -59,6 +59,8 @@ declare global {
     export interface PublicGathererArtifacts {
       /** Console deprecation and intervention warnings logged by Chrome during page load. */
       ConsoleMessages: Crdp.Log.EntryAddedEvent[];
+      /** All the IFrame elements in the page.*/
```

Comment:
```suggestion
      /** All the iframe elements in the page. */
```

lowercase? I always think `IFrame` looks awkward but maybe that's just me :)